Casks/font/font-t/font-tengwar-formal-csur.rb

```diff
@@ -7,9 +7,8 @@
   homepage "https://freetengwar.sourceforge.net/formal.html"
 
   livecheck do
-    url :homepage
-    regex(/href=.*?TengwarFormalCSUR(\d+)\.zip/i)
-    strategy :page_match
+    url "https://sourceforge.net/projects/freetengwar/rss?path=/TengwarFont"
+    regex(%r{url=.*?/TengwarFormalCSURv?(\d+(?:\.\d+)*)\.zip}i)
```

Comment:
```suggestion
    regex(%r{url=.*?/TengwarFormalCSUR[._-]?v?(\d+(?:\.\d+)*)\.zip}i)
```

I sometimes include an optional `[._-]?` delimiter between the software name and version in filenames that don't include one. That is to say, we want this to continue matching if upstream modifies the filename to include a delimiter in the future.